packages/devextreme-scss/scss/widgets/base/chat/layout/chat-confirmationpopup/_index.scss

```diff
@@ -0,0 +1,25 @@
+.dx-chat-delete-confirmation-popup-wrapper {
+  .dx-overlay-content {
+    padding: 16px 24px;
```

Comment:
It's better to override already existed selectors in box solution, instead of override exsting + add the new ones